### ISSUE TITLE
Methods with same name should be located side by side

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#255]
If methods of a class share the same name, these methods - if they share the same accessibility - should be place side by side.
This allows easy finding of such methods.
